README.md

```diff
@@ -51,15 +53,51 @@ conda activate rerunnaro
 ```
 Step 2: Train the PCL component by running the main.py file within the virtual environment. Now, #3, #4 and #5 are commented to be disabled. 
 
+```python
+#1
+preprocess_images(path,params['PCL_ZscoreNormalization'],params['PCL_patch_size'])
+
+#2
+patch_contrastive_learning(path,params)    
+
+#3
+#params = architecture_search(path,params,possible_params)
+
+#4
+#run_NaroNet(path,params)
+
+#5
+#get_BioInsights(path,params)
+```
+
 ```sh
 cd your_path/NaroNet/NaroNet-main/src/main.py
 python3 main.py
 ```
 
-Step 3: PCL inference process. With the trained weights, the model generates the embeddings. Run the main.py file again in the same configuration and virtual environment.
+Step 3: PCL inference process. With the trained weights, the model generates the embeddings. Rerun the main.py file in the same configuration and virtual environment.
 
 Step 4: Run NaroNet and get BioInsights. Here, the commented methods are #1, #2. 
 
+```python
+#1
+#preprocess_images(path,params['PCL_ZscoreNormalization'],params['PCL_patch_size'])
+
+#2
+#patch_contrastive_learning(path,params)    
+
+#3
+params = architecture_search(path,params,possible_params)
+
+#4
+run_NaroNet(path,params)
+
+#5
+get_BioInsights(path,params)
+```
+
+# Exp 3
```

Comment:
You haven't explained experiment 2?

Reply:
Experiment 2 is completely similar to 1, besides the initial preprocessing steps. I can add a short sentence on this. We talked about skipping that from the public repo and I will give you the details on running the pipeline in another private file. Is that fine?

Reply:
Yes please mention that exp2 is similar to 1. Why did we decide again on keeping it private?

Reply:
The dataset used is in-house EMC and will not be made publicly available as far as I know.

Reply:
I meant the methods you used not the dataset

Reply:
The methods will be shared in a private file